src/core/dynamic_batch_scheduler.cc

```diff
@@ -155,6 +155,7 @@ DynamicBatchScheduler::Enqueue(std::unique_ptr<InferenceRequest>& request)
   INFER_TRACE_ACTIVITY(
       request->Trace(), TRITONSERVER_TRACE_QUEUE_START,
       request->QueueStartNs());
+  request->TraceTensor();
```

Comment:
Doesn't TraceTensor need an argument indicating at what point you are collecting the tensor. Here should be at "Queue Start".